scripts: update `drtprod` create drt-large

herkolategan opened the pull request:
Bring the creation in line with what is currently running on cockroach-drt. Uses zone expansion to balance the nodes evenly across regions.

Epic: None
Release Note: None

herkolategan committed:
scripts: update `drtprod` create drt-large

Bring the creation in line with what is currently running on cockroach-drt. Uses
zone expansion to balance the nodes evenly across regions.

Epic: None
Release Note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/123464)
<!-- Reviewable:end -->

itsbilal commented on scripts/drtprod:
```diff
@@ -128,17 +128,21 @@ EOF"
     fi
     case "${2}" in
       "drt-main")
```
nit: Change this to `drt-large` too?

herkolategan replied:
yep, was about to double check this before moving it out of draft, thanks for pointing it out!

itsbilal commented on scripts/drtprod:
```diff
@@ -193,22 +193,26 @@ EOF"
           --gce-managed \
-          --gce-zones "us-east1-b,us-west1-b,europe-west2-b" \
+          --gce-enable-multiple-stores \
+          --gce-zones "northamerica-northeast2-a:2,northamerica-northeast2-b:2,northamerica-northeast2-c:1,europe-central2-a:2,europe-central2-b:2,europe-central2-c:1,northamerica-northeast1-a:2,northamerica-northeast1-b:2,northamerica-northeast1-c:1" \
```
should we swap Warsaw for Columbus?

herkolategan replied:
Yes, I will update it before merging.

itsbilal reviewed (approved):
LGTM

ajstorm reviewed (approved):
LGTM too

herkolategan committed:
scripts: update `drtprod` drt-large region

In an effort to save cost on regional network transfers the Warsaw Europe Zone
will now be switched to Columbus.

Epic: None
Release Note: None

herkolategan commented:
TFTRs!

bors r=itsbilal,ajstorm
